python/nvstrings/nvcategory.py

```diff
@@ -1,3 +1,7 @@
+import weakref
+
+import rmm  # noqa: F401
```

Comment:
This is to make RMM initialize and register a finalization on exit.

Reply:
And the `noqa: F401` is to make `flake8` happy.